### PR TITLE
Reduce aircraft not being targetable at the north edge of the map.

### PR DESCRIPTION
**Pull request by pchote**
This fixes two of the problems contributing to #9552, fixing the exploit where helicopters at the north edge of the map won't be `AutoTarget`ed by AA.  The problem is reduced (but remains) for fixed-wing aircraft; fixing them properly will require the flight code to predict ahead and turn *before* reaching the edge of the map.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Air/Aircraft.cs [hunk 1]
```diff
@@ -228,6 +228,15 @@ public virtual WVec GetRepulsionForce()
 				repulsionForce += GetRepulsionForce(actor);
 			}
 
+			// Actors outside the map bounds an extra nudge towards the center of the map
```
Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: This seems to be missing a word ("get"?).

**Review by reaperrr — APPROVED** [review]

**Comment by pchote** [comment]
Fixed.

**Review by obrakmann — APPROVED** [review]
👍

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/2efba11698d3f696e6e8dc2867ebb4501bec8fd5)

**Pull request merged**